strings/isisogram.go

```diff
@@ -0,0 +1,69 @@
+// Checks if a given string is an isogram.
+// A first-order isogram is a word in which no letter of the alphabet occurs more than once.
+// A second-order isogram is a word in which each letter appears twice.
+// A third-order isogram is a word in which each letter appears three times.
+// wiki: https://en.wikipedia.org/wiki/Heterogram_(literature)#Isograms
+// Author: M3talM0nk3y
+
+package strings
+
+import (
+	"errors"
+	"regexp"
+	"strings"
+)
+
+const (
+	firstOrder  int = 1
+	secondOrder int = 2
+	thirdOrder  int = 3
+)
```

Comment:
The pattern for this is a bit different.

The idea is to create a new type and pass that type and create these constants (that should be exported)

```suggestion
type order int

const (
	First  order = iota +1
	Second
	Third
)
```

The order is in the name of the new type we defined to can easily be inferred. Check this [playgound](https://go.dev/play/p/7dcoMZe0wZz) for more information about this pattern

Reply:
Makes sense. I did have to export the new type as part of the tests. I had initially added an isogram package but it was suggested to move this to the root of the strings package. Let me know what you think.